src/gui/core/event/dispatcher.hpp

```diff
@@ -183,8 +183,8 @@ class tdispatcher
 	 * @param modifier               The SDL key modifiers used.
 	 * @param unicode                The unicode value for the key pressed.
 	 */
-	bool fire(const tevent event,
-			  twidget& target,
+	bool fire(const event_t event,
```

Comment:
There are a lot of event types in the codebase already, please make it clear what type of event it is by the name.